torch/_meta_registrations.py

```diff
@@ -3600,8 +3624,8 @@ def shift_dtype_check(fn_name, self, val):
 @register_meta([aten.__rshift__.Tensor, aten.__rshift__.Scalar])
 def meta_rshifts(self, other):
     shift_dtype_check("rshift", self, other)
-    element_wise = _elementwise_meta(
-        self, type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT
+    element_wise = elementwise_meta(
+        self, type_promotion=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT
```

Comment:
This still looks fishy as we aren't broadcasting or doing type promotion with `other`.

Reply:
Yeah, that doesn't seem right. Let me fix that

Reply:
This hasn't been addressed

Reply:
ugh, sorry, the rebase messed it up. I'll fix it